### DISCUSSION TITLE
Error indicates a problem with the reach compiler

### DISCUSSION THREAD

**Discussion by Thomas-Bibby** [reach-sh/reach-lang#1119] [General]
Got another reach compiler error when trying to compile my Algo contract I get the error below. I have attached the code.
![image](https://user-images.githubusercontent.com/62004466/172739214-44982dc0-fb20-4ef9-aa94-d2e5f765aebf.png)
[index.zip](https://github.com/reach-sh/reach-lang/files/8865755/index.zip)

**Comment by Thomas-Bibby** [comment]
The issue was resolved after I removed half of the parameters. Not sure if you still want this issue noted

**Reply by jeapostrophe** [reply]
Thank you for making the issue. I'll focus on it there.
